aries-site/src/pages/components/grid.mdx

```diff
@@ -13,3 +22,201 @@ import { GridExample } from '../../examples';
 >
   <GridExample />
 </Example>
+
+## Getting to know Grid
+Grid is a powerful tool. It provides tremendous flexibility and control enabling designers and 
+developers at HPE to create optimal user experiences at any screen size or resolution!
```

Comment:
```suggestion
developers at HPE to create optimal user experiences at any screen size or resolution.
```
Since we don't use exclamations anywhere else, this felt a little out of character.

Reply:
but its SO EXCITING!!!!

done.